src/main/scala/org/broadinstitute/dsde/firecloud/service/RegisterService.scala

```diff
@@ -53,10 +53,17 @@ class RegisterService(val rawlsDao: RawlsDAO, val samDao: SamDAO, val thurloeDao
     }
   }
 
+  def generateWelcomeEmail(userInfo: UserInfo): Notification = {
+    userInfo match {
```

Comment:
Could we match on userInfo.googleAccessTokenThroughB2C instead? It will make the pattern matching more explicit and readable

Reply:
Definitely- I'll make that change

Reply:
Done in 2729bf3c7a0d0071a7e8440fda60f590cad3e07f